recipe/meta.yaml

```diff
@@ -1,26 +1,34 @@
-{% set version = "2.6.0" %}
-{% set sha256 = "7c425211a099f6fa6f74037e6e17be58fb5923b0bd11aea745e48ef83c488b49" %}
+{% set version = "3.0.0" %}
 
 package:
   name: singularity
   version: {{ version }}
 
 source:
-  fn: singularity-{{ version }}.tar.gz
-  url: https://github.com/sylabs/singularity/releases/download/{{ version }}/singularity-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - folder: src/github.com/sylabs/singularity
+    url: https://github.com/sylabs/singularity/releases/download/v{{ version }}/singularity-v{{ version }}.tar.gz
+    sha256: 6d54daf74748018082e71774f0998a20d8242b8bda5d2c7d42573242556230a0
+    patches:
+      - 0001-use-c-ldflags-during-mconfig
 
 build:
   number: 1000
   skip: true  # [not linux]
 
 requirements:
   build:
+    - {{ compiler('go') }}
     - {{ compiler('c') }}
+    - make
   host:
+    - pkg-config
+    - libuuid
     - libarchive 3.3.*
```

Comment:
We might want to update this. They added new symbols in 3.3.3. So it may make sense to make that a lower bound. Alternatively we can pin it globally or in the `libarchive` feedstock and then drop the pinning from here.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/161